xbmc/cores/IPlayer.h

```diff
@@ -226,7 +226,9 @@ enum ViewMode {
   ViewModeStretch16x9,
   ViewModeOriginal,
   ViewModeCustom,
-  ViewModeStretch16x9Nonlin
+  ViewModeStretch16x9Nonlin,
+  ViewModeZoom120Width,
```

Comment:
I originally had it that way, but I changed it to 120% first to match the progression of the spinner in the settings (Normal -> Zoom -> Zoom 120% width -> Zoom 110% width. To me this progression made the most sense, so that Normal and Zoom are first and then the 120% width zoom is smaller than full zoom for some content and then the 110% width zoom is smaller than that. Otherwise, it just looks like it flips back and forth when going through the spinner.). I can change it if you really want and no one else objects.